libs/ui/packages/native/src/components/Form/Input/AnimatedInputSelect/Select.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components/native";
+import Flex from "../../../Layout/Flex/index";
+import Text from "../../../Text/index";
+
+import ChevronDown from "@ledgerhq/icons-ui/native/ChevronDown";
+import { TouchableOpacity } from "react-native";
+import { BaseStyledProps } from "src/components/styled";
+
+const Container = styled(Flex)<{ color: string }>`
+  border-left-width: 1px;
+  border-left-color: ${(p) => p.color};
+`;
+
+const StyledTouchableOpacity = styled(TouchableOpacity)<BaseStyledProps>`
+  justify-content: space-between;
+  flex-direction: row;
+  align-items: center;
+  width: 100%;
+  height: auto;
+  padding: 15px 0px 15px 15px;
```

Comment:
Can we use the spaces from the theme instead of an arbitrary value ? spaces[6] is equal to 16px which is close enough imo